code/modules/projectiles/guns/mods/mods.dm

```diff
@@ -884,7 +884,7 @@
 	I.removal_time *= rand(10, 14)/10
 	I.removal_difficulty *= rand(5, 15)/10
 	I.gun_loc_tag = GUN_MECHANISM
-	I.prefix = "feather trigger"
+	I.prefix = "trigger feathered"
```

Comment:
```suggestion
	I.prefix = "feather trigger"
```

Reply:
This one is actually a typo fix, it makes more grammatical sense this way

Reply:
mind fixing them all then?